guidelines/index.html

```diff
@@ -2961,22 +2961,22 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
 </dt>
 <dd>
    
-   <p>少なくとも 18 ポイント、又は 14 ポイントの太字。あるいは、中国語、日本語、及び韓国語 (CJK) のフォントは、それと同等の文字サイズ。
+   <p>少なくとも 18 ポイント、又は 14 ポイントの太字。中国語、日本語、及び韓国語 (CJK) のフォントについては同等のサイズ。
    </p>
    
-   <div class="note" role="note" id="issue-container-generatedID-114"><div role="heading" class="note-title marker" id="h-note-114" aria-level="3"><span>注記 1</span></div><p class="">特別に細い線のフォント、又は文字の形が分かりにくくなるような独特の見た目や特徴のあるフォントは、コントラストが低い場合に特に読みづらい。
+   <div class="note" role="note" id="issue-container-generatedID-114"><div role="heading" class="note-title marker" id="h-note-114" aria-level="3"><span>注記 1</span></div><p class="">ストロークが極端に細いフォント、又は文字の形状に馴染みの少ない独特な特徴を持つフォントは、コントラストが低い場合に特に読みづらくなる。
    </p></div>
    
-   <div class="note" role="note" id="issue-container-generatedID-115"><div role="heading" class="note-title marker" id="h-note-115" aria-level="3"><span>注記 2</span></div><p class="">ここでいう文字サイズは、コンテンツが提供される際のサイズであり、利用者によるサイズ変更は含まれない。
+   <div class="note" role="note" id="issue-container-generatedID-115"><div role="heading" class="note-title marker" id="h-note-115" aria-level="3"><span>注記 2</span></div><p class="">ここでいうフォントサイズは、コンテンツ配信時のサイズであり、利用者によるサイズ変更は含まれない。
```

Comment:
原文はピリオドで区切られて2文になっているようです。

>Font size is the size when the content is delivered. It does not include resizing that may be done by a user.

こうでしょうか。

```suggestion
   <div class="note" role="note" id="issue-container-generatedID-115"><div role="heading" class="note-title marker" id="h-note-115" aria-level="3"><span>注記 2</span></div><p class="">ここでいうフォントサイズは、コンテンツ配信時のサイズである。利用者によるサイズ変更は含まない。
```